backend/volume/src/game/game.service.ts

```diff
@@ -69,8 +69,8 @@ export class CurrentGameState {
 export class GameService {
   constructor(private readonly prismaGameService: PrismaGameService,
               private readonly prismaUserService: PrismaUserService,
-              private readonly gate: GameGateService,) {
-  }
+              @Inject('gameGate') private readonly gameGate: GateService,
```

Comment:
why name no longer gate, is issue ty

Reply:
these get imported by statusService which needs to differentiate between chat- and gameGate, name is kept same for consistency